catroid/src/org/catrobat/catroid/content/Sprite.java

```diff
@@ -302,6 +304,32 @@ public Sprite clone() {
 		return cloneSprite;
 	}
 
+	public Sprite cloneForCloneBrick() {
+		final Sprite cloneSprite = new Sprite();
+
+		cloneSprite.setName(this.getName() + "-c" + StageActivity.getAndIncrementNumberOfClonedSprites());
+		cloneSprite.isClone = true;
+		cloneSprite.isBackpackObject = false;
+
+		//cloneSprite.lookList = this.lookList;
+		cloneSprite.soundList = this.soundList;
```

Comment:
I thought that I can reuse the sounds since the should never be changed, whereas Looks might be changed by some Bricks (color effects like brightness, alpha etc). Additionally, looks are added to the stage as actors, and I was wondering if adding the identical look multiple times might lead to problems, especially when I call ``sprite.look.remove()`` I only want to remove the copy.

But I did not find a reason why I should copy sounds yet, as far as I can see we would only waste memory by loading the same soundFile into memory for each clone.  (Cloning Look objects is not that costly because there is no deep copy of the LookData, only the reference is copied).

But please correct me if I'm wrong. :smile: